diff --git a/node_modules/@opennextjs/cloudflare/dist/cli/build/bundle-server.js b/node_modules/@opennextjs/cloudflare/dist/cli/build/bundle-server.js
index 11b73af..e482ddd 100644
--- a/node_modules/@opennextjs/cloudflare/dist/cli/build/bundle-server.js
+++ b/node_modules/@opennextjs/cloudflare/dist/cli/build/bundle-server.js
@@ -66,6 +66,7 @@ export async function bundleServer(buildOpts, projectOpts) {
         minifySyntax: projectOpts.minify && !debug,
         legalComments: "none",
         metafile: true,
+        sourcemap: true,
         // Next traces files using the default conditions from `nft` (`node`, `require`, `import` and `default`)
         //
         // Because we use the `node` platform for this build, the "module" condition is used when no conditions are defined.
diff --git a/node_modules/@opennextjs/cloudflare/dist/cli/build/open-next/createServerBundle.js b/node_modules/@opennextjs/cloudflare/dist/cli/build/open-next/createServerBundle.js
index a613769..357ba32 100644
--- a/node_modules/@opennextjs/cloudflare/dist/cli/build/open-next/createServerBundle.js
+++ b/node_modules/@opennextjs/cloudflare/dist/cli/build/open-next/createServerBundle.js
@@ -197,6 +197,7 @@ async function generateBundle(name, options, fnOptions, codeCustomization) {
         entryPoints: [path.join(options.openNextDistDir, "adapters", "server-adapter.js")],
         outfile: path.join(outputPath, packagePath, `index.${outfileExt}`),
         external: ["./middleware.mjs"],
+        sourcemap: true,
         banner: {
             js: [
                 `globalThis.monorepoPackagePath = "${normalizePath(packagePath)}";`,
